packages/react/src/floating-ui-react/hooks/useListNavigation.test.tsx

```diff
@@ -1062,6 +1061,13 @@ describe('useListNavigation', () => {
   it('selectedIndex changing does not steal focus', async () => {
     render(<ListboxFocus />);
 
+    // TODO: This feels like a bug. It's the animation frame callback from `enqueueFocus` sometimes
```

Comment:
@michaldudak Something the team may want to look at. I didn't dig all the way, but this feels like a bug in the library. It seems like the intial focus is handled on an animation frame callback and in some runs of the test this callback runs after the click, causing a different element to be focused. Probably never happens in real life but in the tests the click happens in rapid succession to the initial render.